bpf_test.go

```diff
@@ -253,11 +261,105 @@ func checkPinConfigCleanup(t *testing.T, expectedPaths []string) {
 	}
 }
 
+func checkUpdateDeleteElement(t *testing.T, b *elf.Module) {
+	mp := b.Map("dummy_hash")
+	if mp == nil {
+		t.Fatal("unable to find dummy_hash map")
+	}
+
+	key := 1000
+	value := 1000
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_ANY); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_NOEXIST); err == nil {
+		t.Fatal(err)
+	}
+
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_EXIST); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := b.DeleteElement(mp, unsafe.Pointer(&key)); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_EXIST); err == nil {
+		t.Fatal(err)
+	}
+}
+
+func checkLookupElement(t *testing.T, b *elf.Module) {
+	mp := b.Map("dummy_hash")
+	if mp == nil {
+		t.Fatal("unable to find dummy_hash map")
+	}
+
+	key := 2000
+	value := 2000
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_ANY); err != nil {
+		t.Fatal(err)
```

Comment:
Same thing for these errors, if something fails we should be able to figure out what the problem was at a glance.